cocoviewer.py

```diff
@@ -196,11 +195,26 @@ def draw_masks(draw, objects, obj_categories, ignore, alpha):
                     if m_:
                         draw.polygon(m_, outline=fill, fill=fill)
             # TODO: Fix problem with RLE
-            # elif isinstance(m, dict):
-            #     draw.polygon(m['counts'][1:-2], outline=c[-1], fill=fill)
+            elif isinstance(m, dict):
+                mask = rle_to_mask(m['counts'][:-1], m["size"][0], m["size"][1])
+                mask = Image.fromarray(mask)
+                draw.bitmap((0, 0), mask, fill=fill)
+
             else:
                 continue
 
+def rle_to_mask(rle,height,width):
+  rows, cols = height, width
+  rlePairs = np.array(rle).reshape(-1,2)
+  img = np.zeros(rows*cols, dtype=np.uint8)
+  index_offset = 0
+  for index, length in rlePairs:
+    index_offset += index
+    img[index_offset:index_offset+length] = 255
+    index_offset+=length
+  img = img.reshape(cols,rows)
+  img = img.T
+  return img
```

Comment:
```suggestion

def rle_to_mask(rle, height, width):
    rows, cols = height, width
    rle_pairs = np.array(rle).reshape(-1, 2)
    img = np.zeros(rows * cols, dtype=np.uint8)
    index_offset = 0

    for index, length in rle_pairs:
        index_offset += index
        img[index_offset:index_offset + length] = 255
        index_offset += length

    img = img.reshape(cols, rows)
    img = img.T
    return img

```
Python code style: let's use common python coding styles: snake case for var names, 4 spaces for indentation, spacing after `,`, 2 empty lines between high level functions definitions.